admin/modules/config/thread_prefixes.php

```diff
@@ -510,14 +510,65 @@ function checkAction(id)
 
 	$table = new Table;
 	$table->construct_header($lang->prefix);
+	$table->construct_header($lang->forums);
 	$table->construct_header($lang->controls, array('class' => 'align_center', 'colspan' => 2));
 
 	$prefixes = build_prefixes();
+
+	foreach($prefixes as &$prefix)
```

Comment:
`build_prefixes()` could return `false` [ref](https://github.com/mybb/mybb/blob/mybb_1822/inc/functions.php#L3648), so it's better to check if it's an array.